### PR TITLE
SW-2744 Create weight system selector

### PR DESCRIPTION
**Pull request by constanzauanini**
Just adding the dropdown on my account page.
Saving to user preferences will be done separately.

### CODE REVIEW

File: src/components/MyAccount/index.tsx [hunk 1]
```diff
@@ -369,6 +373,26 @@ const MyAccountContent = ({
                 </Grid>
               </>
             )}
+            {weightUnitsEnabled && (
+              <Grid item xs={isMobile ? 12 : 4} sx={{ '&.MuiGrid-item': { paddingTop: theme.spacing(2) } }}>
```
Comment (karthikbtf) [inline src/components/MyAccount/index.tsx:377]: may want a little bit of marginTop in mobile view

**Review by karthikbtf — APPROVED** [review]

File: src/components/MyAccount/index.tsx [hunk 2]
```diff
@@ -369,6 +373,24 @@ const MyAccountContent = ({
                 </Grid>
               </>
             )}
+            {weightUnitsEnabled && (
+              <Grid item xs={isMobile ? 12 : 4} sx={{ '&.MuiGrid-item': { paddingTop: theme.spacing(3) } }}>
```
Comment (karthikbtf) [inline src/components/MyAccount/index.tsx:377]: i think you want margin top 3 in mobile only.. in desktop view it shares same row as time zones and the margin top is not consistent

File: src/components/MyAccount/WeightSystemSelector.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { Dropdown } from '@terraware/web-components';
```
Comment (karthikbtf) [inline src/components/MyAccount/WeightSystemSelector.tsx:1]: move this under `/components`?

File: src/components/MyAccount/index.tsx (hunk 2 above)
Comment (constanzauanini) [reply src/components/MyAccount/index.tsx]: Good catch!

**Review by karthikbtf — APPROVED** [review]

**Review by karthikbtf — APPROVED** [review]
🎉

**Pull request merged**